torchtitan/parallelisms/parallelize_llama.py

```diff
@@ -282,10 +279,9 @@ def apply_ac(model: nn.Module, ac_config):
 def apply_compile(model: nn.Module):
     """Apply torch.compile to each transformer block."""
 
-    # the following flag can be used to to accelarate per-TransformerBlock compilation
-    # TODO(bdhirsh): turning it off because it's currently not working with 2D
+    # the following flag is used to accelarate per-TransformerBlock compilation
     # TODO(anijain): remove it after it's enabled in pytorch by default
-    # torch._dynamo.config.inline_inbuilt_nn_modules = True
+    torch._dynamo.config.inline_inbuilt_nn_modules = True
```

Comment:
Actually... this should be no longer necessary since @anijain2305 recently made it on-by-default for OSS :) https://github.com/pytorch/pytorch/blob/main/torch/_dynamo/config.py#L374

(although I guess since this is off for fbcode we'll still need to manually set this if you want to run in internal)

Reply:
Thanks for the update! I'll remove it then.